.clang-format

```
@@ -0,0 +1,5 @@
+# Copyright cocotb contributors
+# Licensed under the Revised BSD License, see LICENSE for details.
+# SPDX-License-Identifier: BSD-3-Clause
+BasedOnStyle:  Google
+IndentWidth:     4
```

```suggestion
IndentWidth:     4
AccessModifierOffset: -2
```
which is consistent with doubling the indent.

---

Or

```suggestion
IndentWidth:     4
AccessModifierOffset: -4
```
if we want to match what we already have.